data.sql

```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018/11/15', 2, true, 8);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021/01/07', 1, false, 15.04);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017/05/12', 5, true, 11);
+
+/* DAY 2 */
+ALTER TABLE animals ADD COLUMN species text;
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', '2020/02/08', 0, false, -11);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (6, 'Plantnom', '2021/11/15', 2, true, -5.7);
```

Comment:
- [ ] The name of this animal is `Plantmon` and not `Plantnom`. Kindly correct the typographical error made as it could affect query results. 👍